src/AdminConsole/Services/AdminService.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.EntityFrameworkCore;
+using Passwordless.AdminConsole.Db;
+using Passwordless.AdminConsole.Helpers;
+
+namespace Passwordless.AdminConsole.Services;
+
+public class AdminService : IAdminService
+{
+    private readonly IPasswordlessClient _passwordlessClient;
+    private readonly ConsoleDbContext _db;
+    private readonly ILogger<AdminService> _logger;
+    private readonly int _organizationId;
+
+    public AdminService(
+        IPasswordlessClient passwordlessClient,
+        ConsoleDbContext db,
+        IHttpContextAccessor httpContextAccessor,
+        ILogger<AdminService> logger)
+    {
+        _passwordlessClient = passwordlessClient;
+        _db = db;
+        _logger = logger;
+        _organizationId = httpContextAccessor.HttpContext!.User.GetOrgId()!.Value;
+    }
+
+    public async Task<bool> CanDisableMagicLinksAsync()
```

Comment:
Checks whether there are admins without credentials. We could iterate over the `adminconsole` app and list all credentials, but in our hosted environment this could potentially return over >10k records. So this is a lower effort alternative without adding extra endpoints.